run/test/specs/utils/open_app.ts

```diff
@@ -225,21 +287,24 @@ const openiOSApp = async (
 }> => {
   console.warn("openiOSApp");
 
-  // Logging to check that app path is correct
-  console.log(
-    `iOS App Full Path: ${
-      getIosCapabilities(capabilitiesIndex)["alwaysMatch"]["appium:app"]
-    }`
-  );
+  const workerId = parseInt(process.env.MOCHA_WORKER_ID || "0", 10);
```

Comment:
you might want to check for parseInt return as it returns NaN when it cannot be parsed, and I don't think NaN is fasly